src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs

```diff
@@ -94,7 +94,7 @@ public override bool Equals(System.Object obj)
         /// <returns>True if equal GestureInfoType, otherwise false</returns>
         public bool Equals(GestureInfoType other)
         {
-            if ((other == null) || !this.GetType().Equals(other.GetType()))
```

Comment:
Shouldn't we check `(other == null)` condition here anymore?
The original code is that the function returns false when `other` parameter is null value.

Reply:
Since `GestureInfoType` is struct, and ot `INullable`, it cannot be null. (This is what compiler said to me). I just agree with it so remove this null check.

Reply:
Got it. Thank you for explaining it :)